FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_mutual_auth_wolfSSL/README_wolfSSL.md

```diff
@@ -0,0 +1,59 @@
+#  MQTT Mutual Auth wolfSSL Demo
+
+# Overview
+This demo shows an MQTT subscriber / publisher establishing a TLS connection with an MQTT broker to send and receive topics in encrypted form. wolfSSL acts as a TLS library under the FreeRTOSMQTT library.
+A single FreeRTOS task acts as both an MQTT subscriber and an MQTT publisher, sending and receiving topics through the AWS MQTT broker endpoint. The task holds a client certificate and authenticate each other with the MQTT broker.
+
+
+# AWS IoT MQTT Broker Endpoint
+
+wolfSSL provides an AWS IoT MQTT broker endpoint for this demo. The endpoint can be found in
+demo_config.h. Certificates for the tasks and root CA are also provided in the DemoCerts folder. Please note that this endpoint and certificates are provided only for the demonstration or your evaluation purpose only. 

```

Comment:
It may be better for users to setup there own AWS IoT account, so they can see the messages in the cloud. Also someone with malicious intentions can incur a lot of charges on the account that is being presented here. I would recommend removing the credentials that are in this pull request from your AWS account.